docs/index.md

```diff
@@ -114,7 +114,14 @@ First install yotta's dependencies using your system's package manager, for
 example on Debian and Ubuntu:
 
 ```sh
+# install development tool dependencies
 sudo apt-get install python-pip cmake build-essential ninja-build python-dev
+
+# update pip to latest release
+sudo apt-get remove python-pip
+curl -o get-pip.py https://bootstrap.pypa.io/get-pip.py
+sudo python get-pip.py
+hash -r
```

Comment:
Yeah, what can you do - these long haired "believe-in-the-good-of-mankind" open source developers :-).

It's not superbad, as curl verifies at least that the SSL is certified by someone who is allowed to run abitrary code on your machine (AKA your friendly HCHQ / NSA case person et al).

It's not really fixable from the outside (http://pip.readthedocs.org/en/latest/installing.html) - as you usually want the latest version - unless we want to maintain a mirror.

Creating a yotta package for pip would be a catch-22.

Same with using a SHA256 checksum etc - once the file changes we're toast.

The RPM and DEB pakages with ignatures are hopelssly outdated (ubunto, Fedora). Unless you have a better idea I suggest to stick with my suggested patch.